docs/plugins.md

```diff
@@ -78,65 +78,112 @@ Snowpack will automatically call this function to load your plugin. That functio
 For our first example, we’ll look at transforming a file.
 
 ```js
-module.exports = (snowpackConfig, pluginOptions) => ({
-  name: 'my-commenter-plugin',
-  async transform({ fileExt, filePath, contents, isDev }) {
-    if (fileExt === '.js') {
-      return `/* I’m a comment! */ ${contents}`;
+module.exports = function(snowpackConfig, pluginOptions) {
+  return {
+    name: 'my-commenter-plugin',
+    async transform({ fileExt, filePath, contents, isDev }) {
+      if (fileExt === '.js') {
+        return `/* I’m a comment! */ ${contents}`;
+      }
     }
-  }
-})
+  };
+};
 ```
 
-This simple plugin takes all JavaScript files and prepends a simple comment (`/* I’m a comment */`) to the beginning of each file. Even though this is a contrived example, it introduces us to the plugin API.
+The object returned by this function is a **Snowpack Plugin**. A plugin consists of a `name` property and some hooks into the Snowpack lifecycle to customizes your build pipeline or dev environment. In the example above we have:
 
-- The **name** property should be the name of your plugin. This is usually the same as your package name if published to npm.
-- The **transform** method is the function that transforms your file and returns the new file contents to use in the build. Notice that this gets called on every file, so it's up to you to check the file path & extension before performing your transformation.
+- The **name** property: The name of your plugin. This is usually the same as your package name if published to npm.
+- The **transform** method: A function that allows you to transform & modify built files. In this case, we add a simple comment (`/* I’m a comment */`) to the beginning of every JS file in your build.
 
 This covers the basics of single-file transformations. In our next example, we’ll show how to compile a source file and change the file extension in the process.
 
-### File Loading & Building
+### Build From Source
+
+For a more complicated example, we’ll take one input file (`.svelte`) and use it to generate 2 output files (`.js` and `.css`).
```

Comment:
I know this is a late comment but I was just skimming the diff to try and figure out what changed in 2.7.0

This line doesn't seem right, seems like a duplicate from the section below "Multi-File Building", maybe bad copy/paste?

Reply:
ah great catch, just pushed a fix